docs/reference/method/MongoDBModelIndexInfo-isGeoHaystack.txt

```diff
@@ -4,6 +4,9 @@ MongoDB\\Model\\IndexInfo::isGeoHaystack()
 
 .. versionadded:: 1.4
 
+.. deprecated:: 1.16
+   MongoDB 5.0 and later no longer supports geoHaystack indexes.
```

Comment:
The doc was updated in https://github.com/mongodb/mongo-php-library/pull/1085#issue-1732068752 to mention the feature deprecation. I moved this note to a deprecation block.